src/main/java/org/jspecify/annotations/NullMarked.java

```diff
@@ -104,7 +104,8 @@
  *   <li>Applying this annotation to an instance <b>method</b> of a <i>generic</i> class is
  *       acceptable, but is not recommended because it can lead to some confusing situations.
  *   <li>An advantage of Java modules is that you can make a lot of code null-marked with just a
- *       single annotation (before the {@code module} keyword).
+ *       single annotation (before the {@code module} keyword). {@link NullUnmarked} is not
```

Comment:
The text above the list still suggests that NullUnmarked is usable on a module.

"and {@link NullUnmarked}, with one exception" or something?

And then you get to regenerate the Javadoc :)